### PR TITLE
feat: Replace Auction banner with Squad banner

### PR DESCRIPTION
**Pull request by Chef-Cheems**

### CODE REVIEW

File: src/views/Home/components/Banners/PancakeSquadBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    opacity: 1;
+
+    & img {
+      height: 100%;
+    }
+  }
+`
```
Comment (0xCorgi) [inline src/views/Home/components/Banners/PancakeSquadBanner.tsx:82]: It's debatable but I find it more clean to have styled components in a separate `styles` file.
When opening a file reading the render function usually comes first before reading the styles.

File: src/views/Home/components/Banners/PancakeSquadBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Text, Flex, Button, ArrowForwardIcon, Heading } from '@pancakeswap/uikit'
+import { Link } from 'react-router-dom'
+import { useTranslation } from 'contexts/Localization'
+
+const StyledSubheading = styled(Heading)`
+  background: -webkit-linear-gradient(#ffd800, #eb8c00);
```
Comment (0xCorgi) [inline src/views/Home/components/Banners/PancakeSquadBanner.tsx:8]: Is there a reason to use `linear-gradient` with webkit prefix only here ?

File: src/views/Home/components/Banners/PancakeSquadBanner.tsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+      <Inner>
+        <LeftWrapper>
+          <StyledSubheading>PancakeSquad NFT</StyledSubheading>
+          <StyledHeading scale="xl">{t('Coming Oct. 7')}</StyledHeading>
+          <Link to="/pancake-squad">
+            <Button>
+              <Text color="invertedContrast" bold fontSize="16px" mr="4px">
+                {t('Learn more')}
+              </Text>
+              <ArrowForwardIcon color="invertedContrast" />
```
Comment (0xCorgi) [inline src/views/Home/components/Banners/PancakeSquadBanner.tsx:98]: All the colors above seems fixed but this one will change on dark mode, is this ok ?

File: src/views/Home/components/Banners/PancakeSquadBanner.tsx (hunk 1 above)
Comment (Chef-Cheems) [reply src/views/Home/components/Banners/PancakeSquadBanner.tsx:82]: This is copy-paste from FarmAuctionsBanner with tiny css tweaks 😅 
Its gonna be removed completely once Squads are live

File: src/views/Home/components/Banners/PancakeSquadBanner.tsx (hunk 2 above)
Comment (Chef-Cheems) [reply src/views/Home/components/Banners/PancakeSquadBanner.tsx:8]: I think there was a reason, but idk...
> This is copy-paste from FarmAuctionsBanner with tiny css tweaks 😅 
Its gonna be removed completely once Squads are live

File: src/views/Home/components/Banners/PancakeSquadBanner.tsx (hunk 3 above)
Comment (Chef-Cheems) [reply src/views/Home/components/Banners/PancakeSquadBanner.tsx:98]: YOLO
YEET FAST

**Review by 0xCorgi — APPROVED** [review]

**Review by ChefKai — APPROVED** [review]

**Pull request merged**